### DISCUSSION TITLE
Concerns about per-database pricing model

### DISCUSSION THREAD

**Discussion by tasarsu** [planetscale/discussion#447] [Feature Requests] [answered]
Dear PlanetScale Team,

I'm interested in using your database-as-a-service platform for a multi-tenant application that requires separate databases for each tenant. However, I'm concerned about the per-database pricing model and the potential cost implications for my application.

I'm wondering if you would consider offering an alternative pricing model that would better suit the needs of multi-tenant applications. Perhaps a pricing model based on total storage or read/write requests, rather than per-database pricing, could be more cost-effective for applications that require multiple databases.

Alternatively, would it be possible to allow for separate data storage for each tenant while still using a single database schema? This would enable us to use a single database while still keeping tenant data separate and avoiding the costs associated with multiple databases.

I appreciate your consideration of these requests and look forward to hearing your thoughts.

Kind regards

**Comment by fooforge** [comment, marked as answer]
Hey @tasarsu,

Thanks for taking the time to share feedback with us. I appreciate it!

I have answered a similar question in https://github.com/planetscale/discussion/discussions/445#discussioncomment-5674300 last night. The tl;dr is that we're aware that we're not the greatest fit for users with lots of small databases yet, and that it's something we would like to change in the future. There are, however, technical limitations such as Vitess only allowing for one schema per cluster which we have to address first. 

I can't say if or when we're going to get to it yet, but I would recommend keeping an eye on [our changelog](https://planetscale.com/changelog) to be informed about any changes with regards to our pricing and/or database hosting models.

**Comment by tasarsu** [comment]
Thank you @fooforge for your prompt response and your consideration of my feedback.

I appreciate your transparency about the issue and your willingness. I will definitely keep an eye on changelog and hope that a solution can be found for multi-tenant applications.

Regards
